### ISSUE TITLE
Группировка платежей (расходов-поступлений) по статьям бюджета

### ISSUE THREAD

**Issue by jekhor** [jekhor/MVOfund#9]
Реализовать группировку платежей по статьям бюджета (возможно, по ссылке из бюджета, или выбором статьи на странице списка платежей).

**Comment by davojta** [comment]
А есть запрос на такую штуку? чтобы именно статьи бюджета финансировать?
Или это такой способ народного голосования будет?

**Comment by davojta** [comment]
мне кажется, было бы удобная функция платить например 10 уе и распределять уже по проектам + ситуационно на проекты давать

по статьям бюджета довольно тяжело платить, тк непонятно + интерфейс перегрузит
+ что будет если по статье уже 100? по ней уже нельзя будет платить?
и что если по некоторым статьям будет недобор?

**Comment by jekhor** [comment]
Нет, финансируются не отдельные статьи, расходы по отдельным статьям идут.
Ну и если есть поступления не в виде пожертвований (например, спонсорская помощь), то это отдельная статья, да.

**Comment by davojta** [comment]
ага то есть у нас есть 
1) крауд поступления
2) поступления от спонсоров
3) статьи расходов по статьям бюджета 

Соотвественно всё это хочется как то представить. Ясно )
